### DISCUSSION TITLE
Insights - Display custom fields in Directus Dashboards

### DISCUSSION THREAD

**Discussion by igiornotigi** [directus/directus#8902] [Feature Requests]
Hello everybody. I would like to suggest that the custom fields added in the `directus_dashboards` collection be shown in the edit panel of the dasboards so that they can be edited.
In this way it is possible to create rules to show different dasboards to different users. Currently I have succeeded but I have to modify the custum fields that I added directly in the database.
What do you think? Thanks

![image](https://user-images.githubusercontent.com/57590952/137727428-0cee8c91-cfd1-4585-b8d6-24eeb46c8413.png)

![image](https://user-images.githubusercontent.com/57590952/137726292-c67b0e25-5a91-4c37-8792-bf2e752da5ee.png)

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
